monai/visualize/gradient_based.py

```diff
@@ -45,35 +46,38 @@ def backward(ctx, grad_output):
 
 class _GradReLU(torch.nn.Module):
     def forward(self, x: torch.Tensor) -> torch.Tensor:
-        out: torch.Tensor = _AutoGradReLU().apply(x)
+        out: torch.Tensor = _AutoGradReLU.apply(x)
```

Comment:
this modification is to mute the warning message `.<class 'monai.visualize.gradient_based._AutoGradReLU'> should not be instantiated. Methods on autograd functionsare all static, so you should invoke them on the class itself. Instantiating an autograd function will raise an error in a future version of PyTorch.`